collector/collector.go

```diff
@@ -403,6 +404,8 @@ func (c Collector) collect(ch chan<- prometheus.Metric, logger log.Logger, clien
 
 // Collect implements Prometheus.Collector.
 func (c Collector) Collect(ch chan<- prometheus.Metric) {
+	c.metrics.SNMPInflight.Inc()
```

Comment:
This doesn't account for multiple worker concurrency. I think this go in the `scaper` package. Or in the `collect()` internal function?

Reply:
It seems my understanding of "in-flight" might have been insufficient. 
I had assumed that the metric would measure the number of Collectors currently processing, but does it generally include the number of workers as well?

Reply:
Yea, my understanding was more about the number of active SNMP requests. What do you think?

Reply:
OK! I will change it